src/Components/HeaderComponents/styles.tsx

```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+
+import { colors } from '../../styles/colors';
+
+export default StyleSheet.create({
+  allCasesBorder: {
+    paddingRight: 230,
```

Comment:
Change this to a percentage width so it spans the entire screen width. Within it, push the image to the left using flex start.